helpdesk_mgmt/data/helpdesk_data.xml

```diff
@@ -45,7 +45,7 @@
                           <table width="590" border="0" cellpadding="0" bgcolor="#ffffff" style="min-width: 590px; background-color: rgb(255, 255, 255); padding: 20px; border-collapse:separate;">
                             <tbody>
                               <td valign="top" style="font-family:Arial,Helvetica,sans-serif; color: #555; font-size: 14px;">
-                                <p>Hello ${object.user_id.name},</p>
+                                <p>Hello ${object.partner_name and object.partner_name or (object.partner_id.name and (object.partner_id and object.partner_id.name) or '')},</p>
```

Comment:
No need to repeat parts of the expression or useless ones, and the contact name should have priority (you may have updated it according your casing preferences for example):

```suggestion
                                <p>Hello ${object.partner_id.name or object.partner_name  or '')},</p>
```

(the same for both)